setup.py

```diff
@@ -130,6 +130,7 @@
         "tables>=3.7.0",
         "pyarrow",
         "scipy<=1.13.1",
+        "cloudpickle",
```

Comment:
`cloudpickle` would have to be added to `arkouda-env-dev.yml`,  `arkouda-env.yml`, and `pydoc/requirements.txt`.

Reply:
Yes.

Aside, I'm currently working on an alternative implementation that sends LLVM IR and thus doesn't require an embedded Python interpreter on the server side (and hence no cloudpickle on the client side). If successful (it's looking good), that would supersede this prototype implementation. Thanks!